styles/widgets/base/scheduler.less

```diff
@@ -442,6 +445,104 @@
     display: none;
 }
 
+.dx-scheduler-header-panel-cell.dx-scheduler-header-panel-current-time-cell {
+    box-shadow: inset 0px -3px 0px 0px @SCHEDULER_APPOINTMENT_BASE_COLOR;
+    color: @SCHEDULER_APPOINTMENT_BASE_COLOR;
+}
+
+.dx-scheduler-date-time-indicator-all-day {
+    margin-left: @SCHEDULER_LEFT_COLUMN_WIDTH;
+    width: 100%;
+    background-color: @SCHEDULER_TIME_INDICATOR_BACKGROUND_COLOR;
+    position: absolute;
+    pointer-events: none;
+}
+
+.dx-scheduler-date-time-indicator-line {
+        background-color:  @SCHEDULER_APPOINTMENT_BASE_COLOR;
+        position: absolute;
+        box-shadow: @SCHEDULER_TIME_INDICATOR_SHADOW;
+
+        &:before {
+            font-size: @SCHEDULER_TIME_INDICATOR_FONT_SIZE;
+            color: @SCHEDULER_APPOINTMENT_BASE_COLOR;
```

Comment:
Why indicator line has font-size and color settings? Currently, It does not contain any text.

Reply:
But it has a spin-icon